docker_test.go

```diff
@@ -458,21 +458,23 @@ func TestXrayDockerScan(t *testing.T) {
 	initContainerTest(t)
 	initXrayCli()
 	validateXrayVersion(t, scan.DockerScanMinXrayVersion)
-
+	// Create server config to use with the command.
+	createJfrogHomeConfig(t, true)
```

Comment:
The func `createJfrogHomeConfig` changes the JFrog home directory env, we need to restore it at the end.


```
homeEnv := os.Getenv(coreutils.HomeDir)
defer testsutils.SetEnvAndAssert(t, coreutils.HomeDir, homeEnv)
```